### PR TITLE
Project Todo - Julia Ros

### PR DESCRIPTION
**Pull request by julialeonieros**
Link to my deployed project:
https://zealous-ardinghelli-8278dd.netlify.app/

My reflections on how this week's project turned out:
I'm pretty happy with how the app turned out. It looks best on mobile though. 

Things I'd like to have clarified or explained in more detail:
I have problems getting eslint to work (disabled it before and now it doesn't "work"), so there might be some errors/warnings regarding the formatting that I don't see. I tried re-installing it but had to remove it due to a lot of errors, but then I couldn't deploy it because of mismatch on eslint versions. In the end I had to delete my repo and fork it again to be able to deploy it.

### CODE REVIEW

File: code/src/components/AddTodo.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+  @media (min-width: 767px){
+    padding: 20px 48px;
+  }
+  @media (min-width: 1024px){
+    width: 700px;
+    margin: auto;
+    padding: 20px 0;
+  }
+`
```
Comment (Aepera) [inline code/src/components/AddTodo.js:19]: Nice job with using styled components throughout the project 👌

File: code/src/components/AddTodo.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    }
+    dispatch(todos.actions.addTodo(newTodo))
+    setValue('')
+  }
+
+  return (
+    <TodoWrapper>
+      <Form onSubmit={onFormSubmit}>
+        <AddButton type="submit">
+          <i className="fas fa-plus"></i>
```
Comment (Aepera) [inline code/src/components/AddTodo.js:87]: I like the font awesome details

File: code/src/components/TodoList.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                Check all
+              </Button>
+              <Button onClick={() => dispatch(todos.actions.removeChecked())}>
+                Delete checked
+              </Button>
+            </ButtonsContainer>
+          </Footer>
+        </TodolistWrapper>
+        ) : (
+        <EmptyTodoList />
```
Comment (Aepera) [inline code/src/components/TodoList.js:84]: Great use of a ternary operator to display the Empty list component!

File: code/src/reducers/todos.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    addTodo: (store, action) => {
+      store.tasks = [...store.tasks, action.payload]
+    },
+    removeTodo: (store, action) => {
+      const decreasedTasks = store.tasks.filter(todo => todo.id !== action.payload)
+
+      store.tasks = decreasedTasks
+    },
+    checkTodos: (store) => {
+      const checkedTasks = store.tasks.map((todo) => {
```
Comment (Aepera) [inline code/src/reducers/todos.js:33]: Well done with immutable approach 👍

**Review by Aepera — APPROVED** [review]
First impression: I really like the design! Very clean and neat looking, nice colors and color markings. ALso big plus for the coffee animation when all tasks are done! 
Everything is working as supposed to on the app and the responsiveness looks good. The code is clean and easy to read.

File: code/src/components/Todo.js [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+  }
+  &:checked + ${Label}:after {
+    opacity: 1;
+  }
+`
+const Text = styled.p`
+  font-size: 18px;
+  width: 75%;
+  margin-left: 23px;
+  overflow: scroll;
```
Comment (KarolCislowski) [inline code/src/components/Todo.js]: I think `overflow: auto;` would do a trick 
image

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Great app with a nice and clean interface. Please fix the issue with not needed scroll bars 😸

**Review by KarolCislowski — APPROVED** [review]
Great work!

**Pull request closed without merging**